iothub/service/src/Common/EventHubPartitionKeyResolver.cs

```diff
@@ -1,38 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This is not used anywhere and does not seem like something a customer would be using. If anyone has other thoughts, let me know.

Reply:
I do see it being used in a few places. Searching in the Azure Organization, seems like it's being used by edge team:
https://github.com/Azure/iotedge/blob/master/test/modules/TestResultCoordinator/Services/TestResultEventReceivingService.cs

Reply:
Wow! That is some crazy dependency! I will restore it then. We are not using it in any of our code. Does it even make sense for them to use this?

Reply:
Are you sure they are using our implementation or they have their own?

Reply:
So looking at how it's used in their file, seems like it is using the Microsoft.Azure.Devices.Common namespace. However, I am looking at the csproj that is referencing out package, and I see this:
<PackageReference Include="Microsoft.Azure.Devices" Version="1.28.1-NestedEdge" />

This is in https://github.com/Azure/iotedge/blob/df7d717b21eb6ed5b9e8792902c4b985789e9506/test/modules/TestResultCoordinator/TestResultCoordinator.csproj#L23

So maybe they are using their own NuGet? which is a fork of our package?

Reply:
Possible. This is just very weird usage though for a helper. I'll see if I can ping someone and figure this out.

Reply:
I will restore this file till I can figure it out.